### PR TITLE
PR - Styles - Milestone 3 - React Calculator

### PR DESCRIPTION
**Pull request by vieiramanda11**
Hello TSE,

This pull request includes:

  * Style for calculator

Kind regards,

[Amanda](https://github.com/vieiramanda11)

### CODE REVIEW

File: src/styles/index.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
+.app {
+  display: flex;
+  flex-direction: column;
+  width: 700px;
+  margin: 50px auto;
+}
+
+.display {
```
Comment (Oghenebrume50) [inline src/styles/index.css]: display should use  flex scee

File: src/styles/index.css [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  text-align: right;
+  font-size: 80px;
+  opacity: 0.9;
+}
+
+.button-panel {
+  display: flex;
+  flex-direction: column;
+  opacity: 0.9;
+}
```
Comment (Oghenebrume50) [inline src/styles/index.css]: it should have a height of 100px see

File: src/styles/index.css [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+button {
+  height: 100px;
```
Comment (Oghenebrume50) [inline src/styles/index.css]: height would not be needed in you add it to the row

File: src/styles/index.css [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  border: 1px solid gray;
+  color: black;
+  font-weight: bold;
+  font-size: 25px;
+}
```
Comment (Oghenebrume50) [inline src/styles/index.css]: it should take 25% of the width see

Comment (vieiramanda11) [reply src/styles/index.css]: I set the button width in the Button.js component as required in item 5: 
Pass “color” and “wide” props to the Button.

**Pull request merged**